tests/core_widgets/test_presets_widget.py

```diff
@@ -0,0 +1,41 @@
+from micromanager_gui._core_widgets._presets_widget import PresetsWidget
+
+
+def test_preset_widget(qtbot, global_mmcore):
+    for group in global_mmcore.getAvailableConfigGroups():
+        wdg = PresetsWidget(group, text_color="white")
+        qtbot.addWidget(wdg)
+        presets = list(global_mmcore.getAvailableConfigs(group))
+        assert list(wdg.allowedValues()) == presets
+
+        # no need testing the changes of a config group that has <= 1 item
+        if len(presets) <= 1:
+            return
```

Comment:
We may not need a test of them changing. But the code path to create them needs to be executed. They could get a standalone test, or this could be an if <1 and else branch. But needs to be some test of them